recipes/omniorb/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "4.2.1" %}
+{% set release = "2" %}
+{% set sha256 = "9b638c7047a05551c42fe13901194e63b58750d4124654bfa26203d09cb5072d" %}
+
+package:
+  name: omniorb
+  version: {{ version }}
+
+source:
+  fn: omniORB-{{ version }}-{{ release }}.tar.gz
+  url: https://sourceforge.net/projects/omniorb/files/omniORB/omniORB-{{ version }}/omniORB-{{ version }}-{{ release }}.tar.bz2
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - gcc
```

Comment:
I don't get think I get it; if I do that, configure does not find a compiler. 

I'm trying to build on a CentOS 7 machine by the way. Is toolchain something that will only work on conda-forge?